.github/dagger.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "ci",
+  "sdk": "go",
+  "dependencies": [
+    {
+      "name": "gha",
+      "source": "github.com/shykes/gha@b67ffd088b9411c8a01f5d6ca7b83365e3d38484"
```

Comment:
What's the plan for where this lives long-term btw? Could we move it to the `dagger` org or under `modules/`? Not a blocker at all, just curious, personally, I prefer having it "semi-official", with permissions/reviews managed by the same system we use would be good.

Reply:
Yes, we should move it in. There are a few options, I will open a follow-up proposal.